docs/index.md

```diff
@@ -0,0 +1,26 @@
+# Welcome to the documentation for PlantCV-Geopspatial
+
+## Overview
+
+[PlantCV-Geospatial](https://github.com/danforthcenter/plantcv-geospatial) is composed of modular functions in order to be applicable to a 
+variety of plot designs and imaging systems. In the following documentation we will describe use of each function and 
+provide tutorials on how each function is used in the context of an overall image-processing workflow. Development of 
+PlantCV-Geospatial is ongoing---we encourage input from the greater plant phenomics community. Please post questions and comments 
+on the [GitHub issues page](https://github.com/danforthcenter/plantcv-geospatial/issues).
+
+## Getting started
+
+The documentation can be navigated using the sidebar table of contents. Documentation for individual PlantCV-Geospatial functions
+are listed under the headings "PlantCV Namespace". For general information on installation,
+updating, and other questions, see:
+
+* [Installing PlantCV-Geospatial](installation.md)
+
+## Contributing 
+
+If you are interested in contributing to PlantCV-Geospatial, please see:
+
+* [Code of Conduct](CODE_OF_CONDUCT.md)
+* [PlantCV Contributing Guide](https://plantcv.readthedocs.io/en/latest/CONTRIBUTING/)
```

Comment:
I think it should direct to PlantCV-Geospatial's contributing.md, and then in contributing.md, there is a link attached to go to the contributing guide of PlantCV.